src/styles/icons.css

```diff
@@ -13,3 +13,52 @@
 .svg-icon-bg.iconEllipsisHorizontal {
     --bg-icon: url("~@stackoverflow/stacks-icons/src/Icon/EllipsisHorizontal.svg");
 }
+
```

Comment:
Those an overrides on top of an existing override in the custom-components.css file. I would prefer if we would tweak the original override rather than adding one on top.